CHANGELOG.md

```diff
@@ -5,12 +5,17 @@ All notable changes to this project will be documented in this file. See [standa
 ## Unreleased
 
 ### ⚠ BREAKING CHANGES
+* **DataStore/API** category using generated Model classes from Amplify CLI will require the minimum Amplify CLI version of 4.21.4. Current implementations do not expect the field types `embedded(type:)` and `embeddedCollection(of:)` that will break with compile time error. When upgrading Amplify Library, please upgrade Amplify CLI before running `amplify codegen models` to codegen the new `Embeddable` types.
 * **Tools:** Update Amplify tools script to resolve min CLI version for codegen changes ([#554](https://github.com/aws-amplify/amplify-ios/pull/554))
 
 ### Bug Fixes
 
 * **DataStore:** Fixed a DataStore issue where lazy `List<M>` initialization would fail for relationships 3+ levels deep ([#534](https://github.com/aws-amplify/amplify-ios/pull/534))
-* **Tools:** Update Amplify tools script to resolve node correctly when NVM is installed ([#524](https://github.com/aws-amplify/amplify-ios/pull/524))
+* **DataStore/API:** Add Emeddable type to store schema info for custom types ([#539](https://github.com/aws-amplify/amplify-ios/pull/539))
+* **Tools:** Update Amplify tools script to resolve node correctly when NVM is installed ([#535](https://github.com/aws-amplify/amplify-ios/pull/535))
+* **Auth** Signout will completely delete the session in webui ([#542](https://github.com/aws-amplify/amplify-ios/pull/542))
+* **Core** Fix plugin configuration validation ([#543](https://github.com/aws-amplify/amplify-ios/pull/543))
+* **Build** Add common dependency configuration, standardize environment across all modules, podspec source version tag and url ([#538](https://github.com/aws-amplify/amplify-ios/pull/538))
```

Comment:
This should be under a miscellaneous section rather than "Bug Fixes"

Reply:
moved